### DISCUSSION TITLE
Requesting a refund

### DISCUSSION THREAD

**Discussion by bbcenas213** [aclap-dev/video-downloadhelper#1417] [License & Purchase requests]
### What's your issue?

I'd like to request a refund, i had bought the license for Firefox when i meant to purchase for edge, id greatly appreciate it if you could assist with that.

Transaction ID | ch_3QFATyAWbRLTBpbW0wbAW8Fv
Invoice ID | 00493660

### The 10 first digits (only 10) of your license number or email address

0d62d322ae

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
